digitalpy/core/zmanager/controller.py

```diff
@@ -1,9 +1,9 @@
 from abc import ABC
-from digitalpy.routing.action_mapper import ActionMapper
-from digitalpy.config.configuration import Configuration
+from digitalpy.core.zmanager.action_mapper import ActionMapper
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position